hrjobcontract/templates/CRM/Hrjobcontract/Import/Form/MapTable.tpl

```diff
@@ -85,13 +85,12 @@
           </table>
         </div>
         <script type="text/javascript">
-          CRM.$(function($) {
-            {if $mappingDetailsError }
-              $('#saveDetails').show();
-            {else}
-              $('#saveDetails').hide();
-            {/if}
-          });
+          {if $mappingDetailsError }
+            cj('#saveDetails').show();
+          {else}
+            cj('#saveDetails').hide();
```

Comment:
When enclosing that code inside CRM.$ the the import page don't work properly and in the console the following error appear ==> (CRM is not defined ) ... So instead of wasting time on investigating  I just checked the core template files and the CRM.$ was removed from there (in MapTable.tpl ) so I did the same.